packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript

```diff
@@ -1,8 +1,10 @@
 name="brave-browser-nightly-deb"
-version="1.37.44"
+version="1.38.57"
 gives="brave-browser-nightly"
-breaks="${gives}-bin ${gives}-app"
+breaks="${gives}-bin ${gives}-app ${gives}-git"
+replace="${gives}"
 url="https://github.com/brave/brave-browser/releases/download/v${version}/${gives}_${version}_amd64.deb"
 description="The Brave browser is a fast, private and secure web browser for PC, Mac and mobile. This is the nightly build of Brave Browser"
-hash="4b656207fab91977a59a529a5ff2502c5e9f5969838174f11c4dbfeff46e37b8"
-maintainer="Korkmatik <9c54efc1-143b-4f5f-9d2f-f6f189583f0f@aleeas.com>"
+hash="e0a9405379041a8557d80855dd7786ed6fdbb08c9da98b736e11fd0234983afc"
+repology=("project: brave" "visiblename: brave-nightly-bin")
```

Comment:
You might want to Scoop for nightly, they have the latest one

![img](http://0x0.st/oN1r.png)

```suggestion
repology=("project: brave" "visiblename: brave-nightly")
```

Reply:
@Henryws your thoughts?